langchain4j/src/main/java/dev/langchain4j/classification/EmbeddingModelTextClassifier.java

```diff
@@ -6,15 +6,9 @@
 import dev.langchain4j.store.embedding.CosineSimilarity;
 import dev.langchain4j.store.embedding.RelevanceScore;
 
-import java.util.ArrayList;
-import java.util.Collection;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
-
-import static dev.langchain4j.internal.ValidationUtils.ensureBetween;
-import static dev.langchain4j.internal.ValidationUtils.ensureGreaterThanZero;
-import static dev.langchain4j.internal.ValidationUtils.ensureNotNull;
+import java.util.*;
+
+import static dev.langchain4j.internal.ValidationUtils.*;
```

Comment:
nit: could we avoid collapsing imports with stars, and keep each individual import instead?

Reply:
Done.